src/bridge-deployer/src/commands/deploy.rs

```diff
@@ -124,16 +129,45 @@ impl DeployCommands {
         // configure minter
         deployer.configure_minter(bft_bridge).await?;
 
+        let base_side_ids = self
+            .bridge_type
+            .finalize(
+                &self.bft_args,
+                network,
+                deployer.bridge_principal(),
+                pk,
+                &agent,
+            )
+            .await?;
+
         // write canister ids file
         canister_ids.write()?;
 
         info!("Canister deployed successfully");
 
+        println!();
         println!(
-            "Canister {canister_type} deployed with ID {canister_id}",
-            canister_type = self.bridge_type.kind(),
-            canister_id = deployer.bridge_principal(),
+            "Canister {} deployed successfully.",
+            self.bridge_type.kind(),
         );
+        println!("Bridge canister principal: {}", canister_id);
+        println!();
+        println!("Wrapped side BFT bridge: {}", bft_bridge);
+        println!("Wrapped side FeeCharge: {}", fee_charge);
+        println!("Wrapped side bridge address: {}", minter_address);
```

Comment:
should we log all of this data instead?

Reply:
Don't print the empty line please, since it's annoying for the integration tests.